WordPress/Classes/Utility/LocalizableStringsUtility.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LocalizableStringsUtility.swift
+//  WordPress
+//
+//  Created by Evangelos Sismanidis on 29.09.17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import Foundation
+
+enum Strings: String {
+    case title = "Title"
+
+    func loc() -> String {
+        return self.locWithComment("")
```

Comment:
Not really sure if it's a good practise to localize strings with empty comments. Turns out it's really really helpful for our translators!